lib/order.rb

```diff
@@ -0,0 +1,59 @@
+# I would like to be able to select some number of several available dishes
+# I would like to check that the total I have been given matches the sum of the various dishes in my order
+
+require_relative 'menu.rb'
+
+class Order
+
+attr_reader :basket, :balance
+
+  def initialize(menu= Menu.new)
+    # (menu_klass = Menu)
+    # @menu_klass = menu_klass
+    @basket = {}
+    @balance = 0
+  end
+
+  def place_order(item, quantity = 1)
+    fail "please order from menu" if !@menu.public_menu_contents.keys.include?(item)
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/NegatedIf: Favor `unless` over `if` for negative conditions. (https://github.com/bbatsov/ruby-style-guide#unless-for-negatives)